ci-operator/step-registry/azure/bastion/pre/azure-bastion-pre-commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
For the script path, I suggest `ci-operator/step-registry/azure/provision/bastion/azure-bastion-commands.sh`, the similar folder structure and files like what aws is doing in https://github.com/openshift/release/pull/22260/files.